docs/source/algorithms/GetDetectorOffsets-v1.rst

```diff
@@ -18,10 +18,22 @@ defines the offset from the reference spectrum.
 
 The algorithm iterates over each spectrum in the workspace and fits a PeakFunction (default is a
 :ref:`Gaussian <func-Gaussian>` function) to the reference peaks. The fit is used
-to calculate the centre of the fitted peak, and the offset is then
-calculated as:
+to calculate the centre of the fitted peak. Using the fitted peak center,
+there are 3 options for calculating offset, depending on the choice for the "OffsetMode"
+input parameter:
 
-:math:`-peakCentre*step/(dreference+PeakCentre*step)`
+Relative (offset relative to reference position):
+
+:math:`offset = -1*peakCentre*step/(dreference+PeakCentre*step)`
```

Comment:
You probably don't need to include the `-1*`, just do `-peakCentre*...`. Same for the other equations.